opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -63,6 +63,15 @@ struct StackdriverOptions {
   std::unordered_map<std::string, google::api::MonitoredResource>
       per_metric_monitored_resource;
 
+  // Optional: per metric labels to add.
+  //
+  // If the value is empty, the label with the given key will be removed.
+  struct Label {
+    std::string key;
+    std::string value;
+  };
+  std::unordered_map<std::string, Label> per_metric_label;
```

Comment:
This looks great, it could actually avoid lots of copy if some labels remain the same over exporter's lifetime.

One question: when view tag and label map conflict, which one takes precedence?

Reply:
IMO per_metric_label takes precedence, because it's more powerful: you can overwrite and delete columns from your views. (whether doing that is a good idea is a separate discussion)

The comment should be explicit about this.

Reply:
Updated comment.